src/khoj/interface/web/chat.html

```diff
@@ -3,7 +3,7 @@
     <head>
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1.0 maximum-scale=1.0">
-        <meta property="og:image" content="https://khoj-web-bucket.s3.amazonaws.com/khoj_hero.png">
+        <meta property="og:image" content="https://assets.khoj.dev/khoj_hero.png">
```

Comment:
We have some other files that are being loaded through a CDN - `pico.min.css`, `intl-tel-input`, `intlTelInput.min.js` are some that come to mind. We should be able to put these in `assets.khoj.dev` as well. Let's just make sure that we find them all before merging this change.

Reply:
I've done this change for the katex files in https://github.com/khoj-ai/khoj/commit/9ae828cf11156896fc4e37db2ce9add558d260ed

Reply:
pico.min.css is already available locally in code, intl-tel-input files aren't strictly necessary to move to assets.khoj.dev but I'll move them at some point soon